src/components/API/getUserLogin.ts

```diff
@@ -0,0 +1,11 @@
+ import axios from 'axios';
+async function getUserLogin(){ 
+    try{
+    const response = await (await axios.get('http://localhost:3333/loginUser')).data;
```

Comment:
No need for response var here you can do return await (await axios.get('http://localhost:3333/loginUser')).data